Makefile

```diff
@@ -87,13 +100,15 @@ SHELL = /usr/bin/env bash -o pipefail
 # -w /home/go/src/github.com/medik8s/node-maintenance-operator         = working dir
 # -e ...                                                        = some env vars, especially set cache to a user writable dir
 # --entrypoint /bin bash ... -c                                 = run bash -c on start; that means the actual command(s) need be wrapped in double quotes, see e.g. check target which will run: bash -c "make test"
-export DOCKER_GO=docker run --rm -v $$(pwd):/home/go/src/github.com/medik8s/node-maintenance-operator \
-	-u $$(id -u) -w /home/go/src/github.com/medik8s/node-maintenance-operator \
+export DOCKER_GO=docker run --rm -v $$(pwd):/home/go/src/github.com/medik8s/$(OPERATOR_NAME)-operator \
+	-u $$(id -u) -w /home/go/src/github.com/medik8s/$(OPERATOR_NAME)-operator \
 	-e "GOPATH=/go" -e "GOFLAGS=-mod=vendor" -e "XDG_CACHE_HOME=/tmp/.cache" \
 	-e "VERSION=$(VERSION)" -e "IMAGE_REGISTRY=$(IMAGE_REGISTRY)" \
 	--entrypoint /bin/bash golang:$(GO_VERSION) -c
 
-all: build
+.PHONY: all
+all: ## Build and push all the three images to quay (docker, bundle, and catalog).
```

Comment:
Looks like `all` targets have changed.
Can you please explain the logic behind this ?

Reply:
This change would interfere with the default `all`, thus I have created `build-push-iib` for this new logic (to build and push images for testing IIB).